cache-config/testing/ort-tests/tcdata/cdnfederations.go

```diff
@@ -34,10 +34,10 @@ func (r *TCData) CreateTestCDNFederations(t *testing.T) {
 
 		data, _, err := TOSession.CreateCDNFederationByName(f, r.TestData.CDNs[i].Name)
 		if err != nil {
-			t.Errorf("could not POST federations: " + err.Error())
+			t.Errorf("could not POST federations: %v", err)
 		}
 		bytes, _ := json.Marshal(data)
-		t.Logf("POST Response: %s\n", bytes)
+		t.Logf("POST Response: %s", string(bytes))
```

Comment:
Nitpick: if you're using %s anyway, the string cast isn't necessary and is expensive, %s will print the []byte type as a string.